automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Rawls.scala

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.dsde.firecloud.api
+
+import akka.http.scaladsl.model.StatusCodes
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.config.{AuthToken, Config}
+
+/**
+  * Rawls API service client. This should only be used when Orchestration does
+  * not provide a required endpoint. This should primarily be used for admin
+  * functions.
+  */
+object Rawls extends FireCloudClient with LazyLogging {
+
+  private val url = Config.FireCloud.rawlsApiUrl
+
+  object admin {
+    def deleteBillingProject(projectName: String)(implicit token: AuthToken): Unit = {
+      logger.info(s"Deleting billing project: $projectName")
+      deleteRequest(url + s"api/admin/billing/$projectName")
+    }
+
+    def deleteUser(subjectId: String)(implicit token: AuthToken): Unit = {
+      logger.info(s"Deleting user: $subjectId")
+      deleteRequest(url + s"api/admin/user/$subjectId")
+    }
+
+    def doesUserExist(subjectId: String)(implicit token: AuthToken): Option[Boolean] = {
+      getRequest(url + s"api/admin/user/$subjectId").status match {
+        case StatusCodes.OK => Option(true)
+        case StatusCodes.NotFound => Option(false)
+        case _ => None
```

Comment:
Ah the old "troolean".

The only usage I see of this is doing a `.getOrElse(false)` -- maybe just make the method return a `Boolean`?